exporter/azure_monitor/example/main.go

```diff
@@ -5,14 +5,19 @@ import (
 	"context"
 	"log"
 	
+	"go.opencensus.io/exporter/azure_monitor/common"
 	"go.opencensus.io/exporter/azure_monitor"
 	"go.opencensus.io/trace"
 )
 
 func main() {
 	ctx := context.Background()
 
-	exporter, err := azure_monitor.NewAzureTraceExporter("111a0d2f-ab53-4b62-a54f-4722f09fd136")
+	exporter, err := azure_monitor.NewAzureTraceExporter(common.Options{
+		InstrumentationKey: "11111111-1111-1111-1111-111111111111", // add your InstrumentationKey

```

Comment:
I will format this better with spaces.

Reply:
Are you doing it in this PR?

Reply:
No need anymore since the other attributes can be set to default.  There are other files where I need to replace tabs with spaces. I will do that in a separate PR called "Formatting whitespace"